components/delivery-instructions.jsx

```diff
@@ -33,9 +33,9 @@ export function DeliveryInstructions ({
 		{ 'o-forms-input--invalid': hasError },
 	]);
 
-	const maxLengthText = maxlength && country === 'GBR'? ` (Max. ${maxlength} characters)` : '';
+	const maxLengthText = maxlength && `(Max. ${maxlength} characters)`;
 	const extraInstruction = country === 'GBR' ? '' : '\u000a- Special handling i.e. place in plastic bag';
-	const defaultPlaceholder = `Enter instructions${maxLengthText}:\u000a- Door colour, letterbox location\u000a- Placement i.e. letterbox delivery${extraInstruction}` ;
+	const defaultPlaceholder = `Enter instructions ${maxLengthText}:\u000a- Door colour, letterbox location\u000a- Placement i.e. letterbox delivery${extraInstruction}` ;
```

Comment:
nitpicking: If we don't have conditions now, we can move `(Max. ${maxlength} characters)` inside of `defaultPlaceholder ` and avoid creating a variable 😃 
```suggestion
	const defaultPlaceholder = `Enter instructions (Max. ${maxlength} characters):\u000a- Door colour, letterbox location\u000a- Placement i.e. letterbox delivery${extraInstruction}` ;
```